public/app/features/panel/panel_menu.js

```diff
@@ -84,6 +86,11 @@ function (angular, $, _) {
             $panelLinksBtn.toggle(showIcon);
           });
 
+          $scope.$watch('panel.helpText', function(helpText) {
+            var showHelpTooltip = (helpText ? helpText.length > 0 : false) && $scope.panel.title !== '';
+            $panelHelpText.toggle(showHelpTooltip);
```

Comment:
This is needed to see if there is any helptext or not like the drilldown link because if helptext is not present then no need to display the tooltip icon.